src/installer/installer.go

```diff
@@ -227,6 +260,25 @@ func (i *installer) extractIgnitionToFS(ignitionPath string) (err error) {
 	return err
 }
 
+func (i *installer) generateSshKeyPair() error {
+	i.log.Info("Generating new SSH key pair")
+	if _, err := i.ops.ExecCommand(utils.NewLogWriter(i.log), "ssh-keygen", "-q", "-f", "/root/.ssh/id_rsa", "-N", "''"); err != nil {
+		i.log.Error(err)
+		return err
+	}
+	return nil
+}
+
+//FIXME:ybettan: do I need this function? it is directly calling ops.CreateFileFromTemplate
```

Comment:
I would put this directly in the caller. It's just creating extra lines of error checking.

Reply:
Agree